exercises/perfect-numbers/example.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var Divisors = require('./divisors');
+
+var PerfectNumbers = function() {
+
+}
+
+PerfectNumbers.prototype.classify = function(number) {
+
+  var i, sum, result;
+  var divisors = new Divisors();
+
+  // Check if the input is valid
+  if (number <= 0)
+    return 'Classification is only possible for natural numbers.';
+
+  // Factorize the current number.
+  var divsArray = divisors.getDivisors(number);
+
+  // Sum the factors.
+  sum = 0;
+  for (i = 0; i < divsArray.length; i++) {
+    sum = sum + divsArray[i];
+  }
```

Comment:
Just a personal preference, but I thought I would mention the [array reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) method.

You could express this code like so:
```javascript
  // Sum the factors.
  sum = divsArray.reduce((sum, div) => sum += div, 0);
```

I mention that just for fun, not because you need to change it.

You could do something similar in your `Divisors` if you wished.